.storybook/stories/dropdown/dropdown.stories.ts

```diff
@@ -17,7 +17,7 @@ const defaultStory: Story = args => ({
         Dropdown
         <cds-icon shape="angle" direction="down"></cds-icon>
       </button>
-      <clr-dropdown-menu>
+      <clr-dropdown-menu *clrIfOpen="open">
```

Comment:
This is really awesome! I thought we were going to have to write interaction tests that actually click the trigger button to open  the dropdown in order to get these snapshots. This is so much simpler, thanks!